### ISSUE TITLE
No link to dashboard?

### ISSUE THREAD

**Issue by kuntzagk** [irods-contrib/metalnx-web#19]
Hi,

while testing Metalnx we only stumbled by accident we stumbled upon the dashboard (because "Save preferences" opens this)
Could you please add a link to it or open it at login for rodsadmin users?

**Comment by ajguerra** [comment]
Well... I believe you cannot see the dashboard anymore because you're no longer a rodsadmin user (since the preferences page changed that).

**Comment by kuntzagk** [comment]
Even if I change it back there is no link to the dashboard.
Even if I logout and login again there is no link to the dashboard.

Only for the "rods" user the Dashboard link is displayed. Since this user is using STANDAR authentication, it is not available in MetaLnx anymore (would be good to allow STANDAR/PAM in parallel but this is another issue)

**Comment by ajguerra** [comment]
BTW, what's the verison of your iRODS?

**Comment by kuntzagk** [comment]
iRODS version is 4.1.9

**Comment by kuntzagk** [comment]
I did some testing again today. Seems that this problem is connected to switching from STANDARD to PAM authentication.

**Comment by ajguerra** [comment]
So, if you use STANDARD everything is ok? PAM causes the problem, then?

**Comment by kuntzagk** [comment]
I don't know if it was PAM alone or the fact that we installed with STANDARD and switched to PAM later on.
Today I did a fresh install (using the setup script) with STANDARD authentication and created a testuser with rodsadmin type and this user can see the dashboard now. Also issue #18 seems to be gone. But issue #21 still seems there.

Just FYI: in our production environment all users come from company LDAP except the "rods" user which is local.

**Comment by ajguerra** [comment]
I'm gonna need more time to set up a grid with LDAP. I'll take a look at that as soon as I can. It's odd that you see authentication exceptions even if the auth scheme is set to STANDARD. I'll try to mimic that.

**Comment by michael-conway** [comment]
FWIW I experience this issue as well and have a corresponding NIEHS repo issue identified, will update here when I fix this one!

**Comment by michael-conway** [comment]
dashboard is dead for now
